test/mock/index.js

```diff
@@ -804,4 +804,753 @@ exports = module.exports = {
       },
     },
   },
```

Comment:
This is a copy of the mocks above expect that the `data` functions return Promises